src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -71,6 +74,18 @@ private struct Element
         private bool queuedRefresh;
         private MirrorData queuedMirrorData;
 
+        /// <summary>
+        ///     Indicates whether preview should stay open, when mouse leaves control.
+        /// </summary>
+        internal bool StaysOpen;
+
+        private const string baseUrl = @"pack://application:,,,/DynamoCoreWpf;component/UI/Images/";
+
+        private readonly BitmapImage pinIconBitmap =
```

Comment:
@aosyatnik - In order to get rid of unnecessary icons, lets try using FontAwesome icons and varying background colors instead. See below:
![pinnedstate-11](https://cloud.githubusercontent.com/assets/2551261/11125011/a3235624-8935-11e5-9120-4b75bb9a7fc6.png)